src/parser/Tokenizer.h

```diff
@@ -173,63 +173,62 @@ struct TurtleTokenCtre {
   compile time by an infeasible amount ( more than 20 minutes for this file
   alone which made debugging impossible) Thus we will currently only allow
   simple prefixes and emit proper warnings.
-            u8"A-Za-z\\x{00C0}-\\x{00D6}\\x{00D8}-\\x{00F6}\\x{00F8}-"
-            u8"\\x{02FF}"
-            u8"\\x{0370}-"
-            u8"\\x{037D}\\x{037F}-\\x{1FFF}\\x{200C}-\\x{200D}\\x{2070}-\\x{"
-            u8"218F}"
-            u8"\\x{2C00}-"
-            u8"\\x{2FEF}"
-            u8"\\x{3001}-\\x{D7FF}\\x{F900}-\\x{FDCF}\\x{FDF0}-\\x{FFFD}"
-            u8"\\x{00010000}-"
-            u8"\\x{000EFFFF}");
+            "A-Za-z\\x{00C0}-\\x{00D6}\\x{00D8}-\\x{00F6}\\x{00F8}-"
+            "\\x{02FF}"
+            "\\x{0370}-"
+            "\\x{037D}\\x{037F}-\\x{1FFF}\\x{200C}-\\x{200D}\\x{2070}-\\x{"
+            "218F}"
+            "\\x{2C00}-"
+            "\\x{2FEF}"
+            "\\x{3001}-\\x{D7FF}\\x{F900}-\\x{FDCF}\\x{FDF0}-\\x{FFFD}"
+            "\\x{00010000}-"
+            "\\x{000EFFFF}");
             */
   static constexpr auto PnCharsBaseString = fixed_string("A-Za-z");
 
-  static constexpr auto PnCharsUString = PnCharsBaseString + u8"_";
+  static constexpr auto PnCharsUString = PnCharsBaseString + fs("_");
```

Comment:
I read the definition of the `fs` lambda in the code above, but did not understand why it's needed (as opposed to just calling `fixed_string`). Can you explain?

Reply:
Ok
1.  `fs("content")` is equivalent to `ctll::fixed_string("content")` . It is just shorter, which helps in this local case because it is needed a lot, and then the "actual" contents of the string become more visible.

2. There is an overload for `operator+` for `fixed_string` and compile time known `char[]` constants. Thus we do not have to write `fs(...` or `fixed_string` for every little string when concatenating, but only for one of them. In the places where I changed `u8"content"` to 'fs("content")` I could also just have gone `"content"`. Do you want me to make this consistent?

Reply:
Thanks for the explanation.

@1: OK, but then maybe the comment, where the `fs` lambda is defined could be clarified. Right now it says "Is needed for some magical reasons".

@2: What about expressions like `PnCharsUString + fs("\\-0-9")` or `cls(PnCharsBaseString) + fs("(\\.") + ...`, is it needed there? Or is it only needed for the first string in a sum?

Reply:
@1 I will.
@2 Since the `+` is left-to-right associative, it suffices, if the firs or second element of a long sum has the type `fixed_string`. I will adapt this for readability